sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -36,6 +36,10 @@ class InteractiveBrowserCredential(InteractiveCredential):
           authenticate work or school accounts.
     :keyword str client_id: Client ID of the Azure Active Directory application users will sign in to. If
           unspecified, the Azure CLI's ID will be used.
+    :keyword str redirect_uri: a redirect URI for the application identified by `client_id` as configured in Azure
+          Active Directory, for example "http://localhost:8400". This is only required when passing a value for
```

Comment:
Is a `redirect_uri` actually required when a `client_id` is provided? Or would the behavior just be the same as before this change?

Reply:
It's required if the application hasn't registered "http://localhost" or "http://localhost:8400" as a redirect URI.